tests/specs/3way/lobby.spec.ts

```diff
@@ -287,8 +288,9 @@ describe('Lobby', () => {
     });
 
     it('moderator leaves while lobby enabled', async () => {
-        // no moderator switching if jaas is available
-        if (ctx.isJaasAvailable()) {
+        // no moderator switching if jaas is available. TODO: revisit this. why limit a non-jaas test based on whether
+        //  jaas is configured or not?!
```

Comment:
Due to different behavior in production environments compared to the rest (different modules handling moderators).

Reply:
I'd rather keep the comment, because this needs addressing. If a certain test works only against certain environments, we should choose which tests to run where.

Reply:
I've removed the comments.

But I'd like to revisit, as I disagree. These tests have nothing to do with jaas, but their behavior changes based on whether you enable jaas tests or not. There are also cases where we silently skip tests when functionality is not available, which defeats the purpose. Let's discuss offline :)